src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -19,6 +20,19 @@ public class MSBuildAPIUtility
         private const string VERSION_TAG = "Version";
         private const string FRAMEWORK_TAG = "TargetFramework";
         private const string FRAMEWORKS_TAG = "TargetFrameworks";
+        private const string UPDATE_OPERATION = "Update";
+        private const string REMOVE_OPERATION = "Remove";
+
+        public ILogger Logger { get; }
```

Comment:
I don't see a lot of state in this class beyond the logger. Maybe this should be a static class instead and the methods could take the logger as needed.